Source/WebCore/page/FocusController.cpp

```diff
@@ -70,6 +70,21 @@ namespace WebCore {
 
 using namespace HTMLNames;
 
+static bool isOpenPopoverWithInvoker(const Node* node)
+{
+    auto* popover = dynamicDowncast<HTMLElement>(node);
+    return popover && popover->popoverData() && popover->popoverData()->visibilityState() == PopoverVisibilityState::Showing && popover->popoverData()->invoker();
+}
+
+static bool isOpenPopoverInvoker(const Node* node)
```

Comment:
The difference between this function and isOpenPopoverWithInvoker is rather subtle.
Can we come up with more descriptive names?

Reply:
I hope it is more descriptive now that one is more like a cast function.